app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class UserMailer < ActionMailer::Base
+  default :from => "ian.m.y.wong@gmail.com"
```

Comment:
Using your own email. Risky. We should use something like Heroku Postmark. Checkout the add-ons on heroku for email services